org.eclipse.xtext/src/org/eclipse/xtext/preferences/BooleanKey.java

```diff
@@ -9,24 +9,40 @@
 package org.eclipse.xtext.preferences;
 
 /**
+ * A boolean preference key.
+ * 
+ * Valid representations are "true" and "false".
+ * 
  * @author Moritz Eysholdt - Initial contribution and API
+ * 
+ * @since 2.26
```

Comment:
Not exactly true but okay

Reply:
There is no distinction between available since, internal since, public since, ... etc. It's kind of unfortunate.